### PR TITLE
53622: improve bug submission text

### PR DESCRIPTION
**Pull request by Mrt134**
- Add text explaining submitting a ticket can take some time.
- Rework sentence explaining the URL popup.
- HTML build test: no issues.
- Need to port TrueNAS text improvements to the other branches

### CODE REVIEW

File: userguide/system.rst [hunk 1]
```diff
@@ -2333,11 +2333,11 @@ complete these fields:
...
   be taken to reproduce it.
 
-After completing the fields, click the :guilabel:`SUBMIT` button to
-automatically generate and upload the report to
-https://redmine.ixsystems.com/projects/freenas/issues. A pop-up menu
-provides a URL so to view status or add additional information to the
-report.
+Click :guilabel:`SUBMIT` to automatically generate and upload the report
+to https://redmine.ixsystems.com/projects/freenas/issues. This process
+can take several minutes, depending on the debug file size. A pop-up
```
Comment (wblock) [inline userguide/system.rst]: Avoid the halting sentence and giving the user information that doesn't help them:
```suggestion
can take several minutes while information is collected and sent.
```

File: userguide/system.rst [hunk 2]
```diff
@@ -2333,11 +2333,11 @@ complete these fields:
...
+window displays a URL to the new ticket. Use the URL to view or add more
```
Comment (wblock) [inline userguide/system.rst]: New paragraph:
```suggestion

  After the new ticket is created, the URL is shown for viewing or updating with more
```

File: userguide/system.rst [hunk 3]
```diff
@@ -2333,11 +2333,11 @@ complete these fields:
...
+information to the report.
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
information.
```

File: userguide/system.rst [hunk 4]
```diff
@@ -2403,13 +2403,12 @@ To generate a support ticket, fill in the fields:
 * **Attachments** is an optional field where configuration files or
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
#ifdef truenas
* **Attachments** is an optional field where configuration files or
```

File: userguide/system.rst [hunk 5]
```diff
@@ -2403,13 +2403,12 @@ To generate a support ticket, fill in the fields:
...
   screenshots of any errors or tracebacks can be included.
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
  screenshots of any errors or tracebacks can be included.
 #endif truenas
```

File: userguide/system.rst [hunk 6]
```diff
@@ -2403,13 +2403,12 @@ To generate a support ticket, fill in the fields:
...
 
-After completing the fields, click the :guilabel:`Submit` button to
-generate and send the support ticket to iXsystems. A pop-up menu
-provides a clickable URL to view the status of or add additional
-information to that support ticket.
-When not already logged into the
-`iXsystems Support page <https://support.ixsystems.com/>`__, clicking
-this URL prompts for a login, or to register a new login.
+Click :guilabel:`Submit` to generate and send the support ticket to
```
Comment (wblock) [inline userguide/system.rst]: By making the attachments paragraph TN-only, we can move the text from above down here and replace this.  This paragraph is not specific to either FN or TN, so we can factor it out like this.

Please come talk to me about this before doing it to make sure I'm not missing something.

File: userguide/system.rst [hunk 7]
```diff
@@ -2333,11 +2333,14 @@ complete these fields:
...
-https://redmine.ixsystems.com/projects/freenas/issues. A pop-up menu
-provides a URL so to view status or add additional information to the
-report.
+Click :guilabel:`SUBMIT` to automatically generate and upload the report
+to the
+`bug tracker <https://redmine.ixsystems.com/projects/freenas/issues>`__.
+This process can take several minutes while information is collected and
+sent.
+
+After the new ticket is created, the URL is shown for updating with more
```
Comment (wblock) [inline userguide/system.rst]: The other branch version was "viewing or updating".

File: userguide/system.rst [hunk 8]
```diff
@@ -2403,13 +2406,16 @@ To generate a support ticket, fill in the fields:
...
-provides a clickable URL to view the status of or add additional
-information to that support ticket.
-When not already logged into the
-`iXsystems Support page <https://support.ixsystems.com/>`__, clicking
-this URL prompts for a login, or to register a new login.
+Click :guilabel:`Submit` to generate and send the support ticket to
+iXsystems. This process can take several minutes while information is
+collected and sent.
+
+After the new ticket is created, the URL is shown for updating with more
```
Comment (wblock) [inline userguide/system.rst]: So was this one.

File: userguide/system.rst (hunk 7 above)
Comment (Mrt134) [reply userguide/system.rst]: It was, but you suggested changing it to the way it appears now.

Comment (Mrt134) [reply userguide/system.rst]: https://github.com/freenas/freenas-docs/pull/625#discussion_r250245464

Comment (wblock) [reply userguide/system.rst]: Sorry, my mistake.  I have fixed it.

**Pull request merged**